libyul/SideEffects.h

```diff
@@ -30,6 +31,20 @@ namespace solidity::yul
  */
 struct SideEffects
 {
+	/// Corresponds to the effect that a YUL-builtin has on a generic data location (storage, memory
+	/// and other blockchain state).
+	enum Effect
```

Comment:
```suggestion
	enum class Effect
```

Reply:
The enum is already in the scope of SideEffects. If we do this, we'll have to write it as `SideEffects::Effect::Write` instead of `SideEffects::Write`. I prefer the latter.

Reply:
Ok